test/Microsoft.AspNet.Mvc.WebApiCompatShimTest/ApiControllerTest.cs

```diff
@@ -161,17 +162,21 @@ public void ApiController_CreatedAtRoute_Dictionary()
             var controller = new ConcreteApiController();
 
             var product = new Product();
-            var values = new RouteValueDictionary(new { controller = "Products" });
+            var expected = new Dictionary<string, object>
+                {
+                    { "controller", "Products" },
+                };
+            var values = new RouteValueDictionary(expected);
```

Comment:
Because directly asserting with `values` fails.
> Assert.Equal() Failure
Expected: RouteValueDictionary [[controller, Products]]
Actual:   Dictionary`<String, Object>` [[controller, Products]]